public/demo.html

```diff
@@ -10,34 +10,39 @@
     <!-- This demo file is used to test both the production bundles after running yarn build -->
     <h1>Humm widgets cards online</h1>
 
-    <h2>AU default widget</h2>
-    <script src="./humm-widgets-cards-au.umd.js?productPrice=400&merchantId='aaa143af-a96b-4887-bcd2-85960da544ea'"></script>
+    <div
+      class="resizeWidget"
+      style="resize: horizontal; overflow: auto; width: fit-content"
+    >
```

Comment:
horizontally resizable parent div to test responsiveness
![image](https://user-images.githubusercontent.com/63985172/136716497-b9560328-187b-40ef-b5ef-7a27a8cba744.png)